### PR TITLE
remove and ban py.io.BytesIO, py.process, py.path.local.sysfind

### PR DESCRIPTION
**Pull request by asottile**

### CODE REVIEW

**Comment by nicoddemus** [comment]
👍

File: testing/test_capture.py [hunk 1]
```diff
@@ -851,7 +853,7 @@ def test_write_bytes_to_buffer(self):
 
 
 def test_bytes_io():
-    f = py.io.BytesIO()
+    f = io.BytesIO()
```
Comment (thisch) [inline testing/test_capture.py]: This test only tests code from the stdlib and can be removed IMO.

Comment (asottile) [reply testing/test_capture.py]: lol true, want to make a patch for it?

Comment (thisch) [reply testing/test_capture.py]: Sure. I'll create a PR

**Pull request merged**